ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource

```diff
@@ -118,7 +117,7 @@ Fill Data Connection Form
         Click Button    ${action_btn}
     END
     Wait Until Generic Modal Disappears
-    Wait Until Project Is Open    project_title=${project_title}
+    Wait Until Element Is Visible    //table[@data-testid="data-connection-table"]
```

Comment:
Maybe you can create a keyword which wait for a given tab to be open, in that way we won't repeat the xpath in multiple places